### PR TITLE
spring example: controller and JSP

### PR DESCRIPTION
**Pull request by YatKat**
Controllers and JSP

### CODE REVIEW

File: src/main/java/mate/academy/springdemo/service/CompanyServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package mate.academy.springdemo.service;
+
+import mate.academy.springdemo.model.Company;
+import mate.academy.springdemo.repository.CompanyRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+
+@Component
```
Comment (ghost) [inline src/main/java/mate/academy/springdemo/service/CompanyServiceImpl.java:10]: are you sure it's a component?

File: src/main/java/mate/academy/springdemo/service/CompanyServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+public class CompanyServiceImpl implements CompanyService {
+
+    @Autowired
```
Comment (ghost) [inline src/main/java/mate/academy/springdemo/service/CompanyServiceImpl.java:13]: better to autowire in constructor

File: src/main/java/mate/academy/springdemo/service/CustomerServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import mate.academy.springdemo.repository.CustomerRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+
+@Component
+public class CustomerServiceImpl implements CustomerService{
+
+    @Autowired
```
Comment (ghost) [inline src/main/java/mate/academy/springdemo/service/CustomerServiceImpl.java:13]: *

File: src/main/java/mate/academy/springdemo/service/SkillsServiceImpl.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+import mate.academy.springdemo.model.Developer;
+import mate.academy.springdemo.model.Skills;
+import mate.academy.springdemo.repository.SkillsRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+
+@Component
```
Comment (ghost) [inline src/main/java/mate/academy/springdemo/service/SkillsServiceImpl.java:11]: *

File: src/main/java/mate/academy/springdemo/service/SkillsServiceImpl.java [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+public class SkillsServiceImpl implements SkillsService {
+
+    @Autowired
```
Comment (ghost) [inline src/main/java/mate/academy/springdemo/service/SkillsServiceImpl.java:14]: *

File: src/main/java/mate/academy/springdemo/controller/DeveloperController.java [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+import org.springframework.web.servlet.ModelAndView;
+
+import javax.annotation.PostConstruct;
+
+@Controller
+@RequestMapping("/developer")
+public class DeveloperController {
+
+    @Autowired
+    private DeveloperService developerService;
```
Comment (ghost) [inline src/main/java/mate/academy/springdemo/controller/DeveloperController.java:19]: use it in constructor

File: src/main/java/mate/academy/springdemo/model/Company.java [hunk 7]
```diff
@@ -0,0 +1,24 @@
+package mate.academy.springdemo.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
```
Comment (ghost) [inline src/main/java/mate/academy/springdemo/model/Company.java]: fix import

File: src/main/java/mate/academy/springdemo/model/Customer.java [hunk 8]
```diff
@@ -0,0 +1,21 @@
+package mate.academy.springdemo.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
```
Comment (ghost) [inline src/main/java/mate/academy/springdemo/model/Customer.java]: *

File: src/main/java/mate/academy/springdemo/model/Developer.java [hunk 9]
```diff
@@ -0,0 +1,26 @@
+package mate.academy.springdemo.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
```
Comment (ghost) [inline src/main/java/mate/academy/springdemo/model/Developer.java]: *

File: src/main/java/mate/academy/springdemo/model/Project.java [hunk 10]
```diff
@@ -0,0 +1,27 @@
+package mate.academy.springdemo.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
```
Comment (ghost) [inline src/main/java/mate/academy/springdemo/model/Project.java]: *

File: src/main/java/mate/academy/springdemo/model/Skills.java [hunk 11]
```diff
@@ -0,0 +1,22 @@
+package mate.academy.springdemo.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
```
Comment (ghost) [inline src/main/java/mate/academy/springdemo/model/Skills.java]: *

File: src/main/java/mate/academy/springdemo/service/CompanyServiceImpl.java (hunk 1 above)
Comment (YatKat) [reply src/main/java/mate/academy/springdemo/service/CompanyServiceImpl.java:10]: Component and Service annotations are similar (written from Bogdan's app example)

**Pull request merged**